lib/packages/docutils/rst.nim

```diff
@@ -312,6 +312,14 @@ proc defaultMsgHandler*(filename: string, line, col: int, msgkind: MsgKind,
   if mc == mcError: raise newException(EParseError, message)
   else: writeLine(stdout, message)
 
+proc noExceptionMsgHandler*(filename: string, line, col: int, msgkind: MsgKind,
+                      arg: string) {.procvar.} =
+  let mc = msgkind.whichMsgClass
+  let a = messages[msgkind] % arg
+  let message = "$1($2, $3) $4: $5" % [filename, $line, $col, $mc, a]
```

Comment:
import std/miscdollars
and use `toLocation` which I've added recently as the single place where location formatting is defined (to allow easy override)

Reply:
Alright. Should I change the proc above as well?

Reply:
somehow I missed that; yes por favor! (or in a different "cleanup" PR depending on whether this is staling or not)